integrationtests/mayavi/test_mlab_savefig.py

```diff
@@ -0,0 +1,95 @@
+import os
+import shutil
+import unittest
+import tempfile
+
+import numpy
+from PIL import Image
```

Comment:
Yes you're right.  This was copied from related PRs; I just removed the import here.